src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {

```

Comment:
According to the checklist, you should not use raw types. The `MachineProducer` interface should be parameterized with a type that extends `Machine`. However, in this case, you have correctly parameterized `MachineProducer<Truck>`, so there are no issues with raw types here.